xlators/debug/io-stats/src/io-stats.c

```diff
@@ -1754,13 +1754,13 @@ int
 update_ios_latency(struct ios_conf *conf, call_frame_t *frame,
                    glusterfs_fop_t op)
 {
-    double elapsed;
+    uint64_t elapsed;
     struct timespec *begin, *end;
 
     begin = &frame->begin;
     end = &frame->end;
 
-    elapsed = gf_tsdiff(begin, end) / 1000.0;
+    elapsed = gf_tsdiff(begin, end);
```

Comment:
collect_ios_latency_sample() is called with elapsed which is now uint64_t - but collect_ios_latency_sample() expects a double?

Reply:
changed to uint64_t

Reply:
changed to uint64_t